sqlmesh/core/model/definition.py

```diff
@@ -12,7 +12,7 @@
 import pandas as pd
 import numpy as np
 from astor import to_source
-from pydantic import Field
+from pydantic import Field, model_validator
```

Comment:
Wrong import for `model_validator`. You need `from sqlmesh.utils.pydantic import ` one